app/views/conference/_venue.html.haml

```diff
@@ -11,7 +11,15 @@
           .thumbnail
             .flexvideo{ id: "resource-content-#{@conference.venue.commercial.id}"}
               = render partial: 'shared/media_item', locals: { commercial: @conference.venue.commercial }
-    
+      - elsif @conference.venue.picture.present?
+        .col-md-5
+          .thumbnail
+            = image_tag @conference.venue.picture.url, alt: "Venue", size: "560x315"
```

Comment:
Please do not hardcode sizes. Use [bootstrap image classes](http://getbootstrap.com/css/#images)